frontend/src/routes/Infrastructure/Clusters/DiscoveredClusters/DiscoveryConfig/DiscoveryConfig.tsx

```diff
@@ -529,6 +532,48 @@ export function DiscoveryConfigPageContent(props: {
               </SelectOption>
             ))}
           </AcmMultiSelect>
+          <AcmMultiSelect
+            id="discoveryClusterTypes"
+            label={t('discoveryConfig.ClusterTypes.label')}
+            labelHelp={t('discoveryConfig.ClusterTypes.labelHelp')}
+            value={discoveryConfig?.spec?.filters?.clusterTypes}
+            placeholder={t('discoveryConfig.ClusterTypes.placeholder')}
+            onChange={(clusterTypes) => {
+              updateDiscoveryConfig((discoveryConfig) => {
+                if (!discoveryConfig.spec.filters) {
+                  discoveryConfig.spec.filters = {}
+                }
```

Comment:
I just learned about nullish coalescing assignment (`??=`) from Sonar the other day... might as well use it here too :)